spec/01_calculator_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Specification for RubyMonk Primer Problem 1 Calculator class
+require_relative '../01_calculator'
+
+RSpec.describe Calculator do
+  subject(:calculator) { described_class.new }
+
+  describe '#add' do
+    it 'adding 2 and 3 gives 5' do
```

Comment:
Personally, not a fan of hardcoding values. I'd do something like
```ruby
describe '#add' do
  it 'adds two numbers' do
    first = Faker::Number.digit
    second = Faker::Number.digit

   expect(calculator.add(first, second)).to eq(first + second)
```

Ideally it'd also include a random chance for a negative number

Reply:
I like that technique and i would have done different tests myself but my intent was to reproduce the original tests, that said, i'm going to add this as an additional test to codify this technique

Reply:
another note that this use of hardcoded is an example of "Specification By Example" which is a good technique to provide human comprehension of abstract concepts through understandable concrete examples.  I'd argue that addition is  not that abstract, so I prefer your more general suggestion.

Reply:
:white_check_mark: